pkg/infra/iac2/templates_compiler_test.go

```diff
@@ -322,7 +322,7 @@ func Test_handleIaCValue(t *testing.T) {
 			resourceVarNamesById: map[core.ResourceId]string{
 				{Provider: "aws", Type: "availability_zones", Name: "AvailabilityZones"}: "azs",
 			},
-			want: "availabilityZones.names[2]",
+			want: "azs.names[2]",
```

Comment:
What changed to require this to change? I don't see any changes to templates_compiler so I'm not sure.

Reply:
so it should always pull the string name in the map that corresponds to the resource id. To be honest, im not sure how this test was ever passing. If you look at the remainder of the tests they follow the same convention. So i dont have a good answer as to why the old way was working